controllers/kafkatopic_controller.go

```diff
@@ -32,10 +35,18 @@ import (
 	"github.com/banzaicloud/koperator/pkg/k8sutil"
 	"github.com/banzaicloud/koperator/pkg/kafkaclient"
 	"github.com/banzaicloud/koperator/pkg/util"
+	"github.com/banzaicloud/koperator/pkg/webhooks"
 )
 
 var topicFinalizer = "finalizer.kafkatopics.kafka.banzaicloud.io"
 
+func isTopicManagedByKoperator(topic metav1.Object) bool {
+	if manager, ok := topic.GetAnnotations()[webhooks.TopicManagedByAnnotationKey]; !ok || strings.ToLower(manager) == webhooks.TopicManagedByKoperatorAnnotationValue {
+		return true
```

Comment:
**Optional**: We **MAY** return the second condition (managed by annotation comparison) directly here without extra conditional to make it more readable.
(Can't give suggestion due to the condition recommendation I made which needs to be cleared up first.)